### PR TITLE
Add Routing 

### PR DESCRIPTION
**Pull request by RakshaPawar108**
- Added routing to all pages 
- Changed anchor tag to links
- Refactored code to export pages and components from single file in respective folders
react-ecomm-routing

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+import { Navbar, Footer } from "./components/index";
+import {
+  Home,
+  Cart,
+  Login,
+  Signup,
+  Logout,
+  ProductListing,
+  Wishlist,
+} from "./pages/index";
```
Comment (AtulPant2704) [inline src/App.jsx]: You don't need to write index over here, just wrting `./pages` would also work, it is the advantage of using index.js for exports, React automatically detects that.

**Review by AtulPant2704 — APPROVED** [review]
In your Home.jsx you have forgotten to add `<Link/>`to `Order Now` bytton, you can do that too. Rest all looks good. Code is very well divided into components which make code easier to review.

**Review by RahullRawat — APPROVED** [review]

**Review by AnkurChunekar — APPROVED** [review]
Good job @RakshaPawar108 , code looks neat and clean and is well indented.

File: src/App.jsx (hunk 1 above)
Comment (RakshaPawar108) [reply src/App.jsx]: Thank you for explaining that. Made the changes!

**Comment by RakshaPawar108** [comment]
> In your Home.jsx you have forgotten to add `<Link/>`to `Order Now` bytton, you can do that too. Rest all looks good. Code is very well divided into components which make code easier to review.

Added the link to products page. Thanks for pointing it out

**Pull request merged**